packages/coordinator/src/screens/manageAppointmentsScreen/ManageAppointmentsTableConfig.tsx

```diff
@@ -29,11 +31,35 @@ export const GetExpandedColumns = (
     hideIfNoData: true,
   },
   {
-    cellRenderer: ({ booked, bookingTimeMillis }) => {
-      if (!booked || !bookingTimeMillis) {
-        return "אין רישום";
+    cellRenderer: ({ booked, bookingTimeMillis, recentChangeType }) => {
+      let bookingDate = "אין רישום";
+      if (booked && bookingTimeMillis) {
+        bookingDate =
+          "נקבע בתאריך " + DateUtils.ToDateString(bookingTimeMillis);
       }
-      return "נקבע בתאריך " + DateUtils.ToDateString(bookingTimeMillis);
+      const chipLabel =
+        recentChangeType === BookingChange.CANCELLED ? "בוטל" : "נקבע";
+      return (
```

Comment:
Should this just be a new column?

This way we could possibly even sort by it.

WDYT?

Reply:
I agree, it might be more convenient